README.md

```diff
@@ -62,14 +62,12 @@ async function handleEvent(event) {
     try {
       return await getAssetFromKV(event)
     } catch (e) {
-        switch (typeof resp) {
-          case NotFoundError: 
-            //..
-          case MethodNotAllowedError:
-            // ...
-          default:
-            return new Response("An unexpected error occurred", { status: 500 })
-        }
+      if (e instanceof NotFoundError) {
```

Comment:
The correct case should be:

```js
import { getAssetFromKV } from '@cloudflare/kv-asset-handler'
import { NotFoundError, MethodNotAllowedError } from '@cloudflare/kv-asset-handler/dist/error'
```

And `error.js` should be export as well. Otherwise the `instanceof` won't work.

Reply:
I'm not sure if `/dist/error` is intended to be imported from outside the package. I think we need to re-export the error types from `index.js`.